python/GafferUI/AnimationEditor.py

```diff
@@ -870,4 +899,186 @@ def __checkBoxStateForTieMode( self, mode ) :
 		commonMode = self.getTieModeForSelectedKeys()
 		return None if commonMode is None else commonMode == mode
 
+# Private implementation - curve widget
+class _CurveWidget( GafferUI.GridContainer ) :
```

Comment:
I can't comment on line 144 where I want to, but we recently added the ability to have customisable columns in the PathListingWidget. We could use that to put a little colour swatch next to the name of each curve, to provide a visual association between the listing and the curve gadget.

Reply:
Yes, this would be my preference. I did it this way to keep the initial PR simple but I agree that a small colour swatch in the PathListingWidget next to the name of the driven plug would be great.

Reply:
I have this working in my branch, however the display transform seems to get "baked" into the colour shown in the swatch so it does not match the colour of the curves as drawn in the gadget. I tried applying the display transform to the curve colour before drawing in the gadget, bu the display transform code is in python which it seems I cannot call from c++ ui code. The colour column does look nice.